### DISCUSSION TITLE
Set the first image when opening the light box

### DISCUSSION THREAD

**Discussion by marvthedev** [igordanchenko/yet-another-react-lightbox#259] [Q&A] [answered]
Is it possible to set which image is viewed first when the lightbox opens up? I want to be able to click on a thumbnail image outside the lightbox when you click on it, the lightbox opens up and displays that image.

**Comment by ash67** [comment, marked as answer]
I think you can add `index` property to the Lightbox component to open a specific slide when the lightbox opens.

```jsx
<Lightbox 
  index={thumbnailImageSlideIndex}
  // ...
/>
```
